src/Uno.UI/UI/Xaml/Controls/Slider/Slider.mux.cs

```diff
@@ -1141,7 +1141,7 @@ private void UpdateTrackLayout()
 					}
 				}
 
-				if (_tpElementHorizontalThumb != null && isThumbToolTipEnabled)
```

Comment:
This code is from WinUI sources. It shouldn't be changed.

Reply:
Where can I find this part of WinUI sources? Isn't it https://github.com/microsoft/microsoft-ui-xaml ?

Reply:
@TopProgrammer77 https://github.com/microsoft/microsoft-ui-xaml/blob/7198c4e69019516fb08ecb3937492ffff5fe9702/dxaml/xcp/dxaml/lib/Slider_Partial.cpp#L1427